AGPs/AGP-1Hive-Sponsorship.md

```diff
@@ -0,0 +1,61 @@
+---
+AGP: TBD
+Title: 1Hive Sponsorship Proposal 
+Authors: 1Hive Team (@1hive on keybase), burrrata (@burrrata), lkngtn (@lkngtn)
+Status: Draft
+Track: Finance
+Created: 2019-09-24
+---
+
+# AGP-TBD: 1Hive Sponsorship Proposal
+
+### Summary
+
+[1Hive](https://1hive.org) is a DAO operating on Aragon. We have multiple initiatives in progress, all of which aim to make Aragon more useful for everyone. Our dream is to support the commons so that open source communities can thrive.
+
+We are a relatively young DAO that only came into existence 4 months ago. In that short time we have initiated several projects, one of which has gone on to receive a nest grant. 
+- [Dandelion](https://1hive.org/blog/2019/08/31/dandelion-overview) - MolochDAO on Aragon ([Nest grantee](https://github.com/aragon/nest/issues/168)).
+- [Apiary](https://apiary.1hive.org) - Like a block explorer, but for Aragon DAOs.
+- [Flora](https://devpost.com/software/flora-sidechain-discount-transactions-q8usv3) - Fast, cheap, and secure sidechain for Aragon DAOs.
+- [Conviction Voting](https://forum.aragon.org/t/conviction-voting-on-aragon-cf-dao-proposal/1195) - Continuous signaling of DAOs sentiment for proposal funding.
+- [Credao](https://devpost.com/software/credao) - Meritocratic reputation for any DAO or community.
+
+These projects are here today because of the 1Hive community. We have taken great care to design, build, and maintain our DAO so that projects like these are possible. We would like to request a sponsorship from Aragon to support the 1Hive DAO. 
+
+This sponsorship is not a request for funding of any of the individual projects listed above. These have been incubated by the 1Hive community and will request additional funding individually when they are ready. We chose this method because want to give the Aragon community more discretion over which projects to support. Our projects will be presented a la carte for ANT holders to review and support individually vs as an all inclusive prix fixe menu. 
+
+This proposal is a sponsorship to allow us to keep growing the 1Hive DAO, reward community members, and incubate more [early ideas](https://github.com/1Hive/ideas/issues). We feel that this is a happy balance between a core team focused on executing a vision and individual projects that can operate with autonomy. 
+
+## Address of the transfer recipient
+
+1Hive Mainnet DAO Agent Address: `address TBD`
```

Comment:
Please input the DAO address or the one person or entity that will provide the DAO address

Reply:
Thank you for catching that. Updated AGP PR accordingly.